app/src/main/java/org/openobservatory/ooniprobe/activity/RunningActivity.java

```diff
@@ -22,6 +22,7 @@
 import com.airbnb.lottie.LottieAnimationView;
 
 import org.openobservatory.ooniprobe.R;
+import org.openobservatory.ooniprobe.common.PreferenceManager;
```

Comment:
Why do you need to import `PreferenceManager` here? As far as I can tell, you're not using it?

Reply:
Removed